hikari/presences.py

```diff
@@ -118,11 +121,21 @@ class ActivityParty:
     """Maximum size of this party, if applicable."""
 
 
+_DYNAMIC_URLS = {
+    "mp": "https://media.discordapp.net/{}",
+    "spotify": "https://i.scdn.co/image/{}",
+    "twitch": "https://static-cdn.jtvnw.net/previews-ttv/live_user_{}.png",
+    "youtube": "https://i.ytimg.com/vi/{}/hqdefault_live.jpg",
+}
+
+
 @attr_extensions.with_copy
 @attr.define(hash=False, kw_only=True, weakref_slot=False)
 class ActivityAssets:
     """Used to represent possible assets for an activity."""
 
+    _application_id: typing.Optional[snowflakes.Snowflake] = attr.field(repr=False)
+
     large_image: typing.Optional[str] = attr.field(repr=False)
     """The ID of the asset's large image, if set."""
```

Comment:
Document here that these may be CDN hashes or may be identifiers pointing towards 3rd party sites